doc/contributing.md

```diff
@@ -1 +1,54 @@
 # Contributing
+Thank you for interest in contributing to LivePeer ecosystem.
+
+# Working on Go code
+If your goal is making changes solely to Go part of the codebase, just follow [Requirements](https://github.com/livepeer/lpms/#requirements) section and make sure you have all dependencies to build Go scripts successfully. Debugging with Delve, GDB, or, visually, with IDE (e.g. JetBrains GoLand, Microsoft VS Code) should work fine.
+
+# Working on CGO interface and FFMpeg code
```

Comment:
We can probably add a short note on what works and doesn't with GDB/Delve.

For example with GDB, I've been able to set breakpoints in Go code, LPMS C code by default, and FFmpeg C code with the build steps given here.

But manually `step`ping from Go -> C boundary doesn't work, and `bt` stack traces are also limited to the language boundary.

Feel free to add if it works better with Delve, or an IDE.

Reply:
Done.